server/src/main/java/com/testsigma/service/TestCaseService.java

```diff
@@ -535,4 +539,16 @@ void updateImportedId(TestCase testCase, TestCase previous, BackupDTO importDTO)
   public void handlePreRequisiteChange(TestCase testCase) {
     this.testSuiteService.handlePreRequisiteChange(testCase);
   }
+
+
+  public void removeStepGroup(Long testCaseId) throws Exception {
```

Comment:
@nixonakash01 Where is this function being called from ? Also, note that after deletion of the step containing a step group, you have to change the position of the steps that come after it

Reply:
@tarun-testsigma The function is not used. I have committed the updated code. The position of the steps after deletion is changed as expected.